roles/configure_ocp_registry_ca/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Update ca bundle
+  become: true
+  ansible.builtin.command:
+    cmd: update-ca-trust extract
+
+- name: Add CAs into OCP registry
+  when:
+    - cifmw_openshift_kubeconfig is defined
+    - cifmw_configure_ocp_registry_ca_registry_to_add is defined
+    - cifmw_path is defined
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    _config_map_data: "{{ cifmw_configure_ocp_registry_ca_registry_to_add }}={{ cifmw_configure_ocp_registry_ca_bundle_path }}"
+  block:
+    - name: Create config map with registry CAs
+      ansible.builtin.command:
+        cmd: >-
+          oc create configmap registry-cas -n openshift-config
+          --from-file="{{ _config_map_data }}"
+
+    - name: Install Red Hat CA for pulling images from internal registry
```

Comment:
While the command used here is fine the task will be flagged as changed always. Can we make it idempotent or just skip it if the CM is already added?